### ISSUE TITLE
Chat Commands duplicate for every player connected

### ISSUE THREAD

**Issue by dancore-san** [sousou63/DarkRP-old#128]

**Comment by QueenPM** [comment]
How about we add an additional parameter/config for commands called ClientOnly, a bool.

If set to true, it would only exectute the command for the client and stop execution for everyone else if they arnt the one who called it?

**Comment by dancore-san** [comment]
I 100% think this is the best route, when I tried fixing it, I created a isBroadcast bool in Command.cs. It worked, the problem was that the broadcasted commands were still duplicating.

**Comment by SebbeJohansson** [comment]
@QueenPM is there a reason why we want the execution of the command to be broadcast? Wouldnt each command broadcast the result?

**Comment by SebbeJohansson** [comment]
I think i have foudn a solution for this. There is an option for the broadcast property to only run on the host.

![image](https://github.com/user-attachments/assets/0401ea10-240c-4508-94f4-c34a6863928f)

**Comment by QueenPM** [comment]
> @QueenPM is there a reason why we want the execution of the command to be broadcast? Wouldnt each command broadcast the result?
> 

Yes this was a quick fix for some broken commands. I think ideally it shouldnt be broadcast and only some parts of specific commands should be

**Comment by SebbeJohansson** [comment]
@QueenPM lets communicate on discord about this.
I made a temporary PR that should at least fix it for now. https://github.com/sousou63/DarkRP/pull/155
